wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -173,23 +173,23 @@ As announced in a [tweet by peppy](https://twitter.com/ppy/status/84802152566384
 
 ### Rotating osu! cookie
 
-On 1 April 2018 the [osu! cookie](/wiki/Brand_identity_guidelines) on the main menu screen would slowly rotate clockwise as time passed, and the cookie in the song selection screen would slowly rotate anticlockwise. Hovering over these cookies would enlarge them just as usual, but would also cause them to rotate faster [[62,63,64,65,66,67]][r].
+On 1 April 2018 the [osu! cookie](/wiki/Brand_identity_guidelines) on the main menu screen would slowly rotate clockwise as time passed, and the cookie in the song selection screen would slowly rotate anticlockwise. Hovering over these cookies would enlarge them just as usual, but would also cause them to rotate faster.[^osu-cookie-web-reddit][^osu-cookie-web-reddit-2][^osu-cookie-web-reddit-3][^osu-cookie-web-forums][^osu-cookie-web-forums-2][^osu-cookie-web-wiki]
```

Comment:
the stuff in https://osu.ppy.sh/wiki/en/History_of_osu%21/2018#april should be moved here

Reply:
you mean the images? because the paragraphs are nearly identical

Reply:
yeah everything. it's an event of only one day and fits much better in this article

Reply:
I'd love to do that in a separate PR

Reply:
ok then